### DISCUSSION TITLE
display images in required field

### DISCUSSION THREAD

**Discussion by ahmadhassan2090** [rfennell/AzurePipelines#1970] [Q&A]
Is there are any way to display images in required field when generating release note ?
![image](https://github.com/user-attachments/assets/0ee9bf37-508e-4fda-bbd5-0c946df0ec4a)

**Comment by rfennell** [comment]
You can add anything you like to your Handlebars template, so yes images are possible. It is just a case of generating the correct image URL, and the image file being in a location that can be accessed by whoever has is reading the release notes
